server/routes/search/searchController.ts

```diff
@@ -58,6 +66,7 @@ export default class SearchController {
     const pagedResults: PagedModelPrisonerSearchDto = await this.doSearch(req, res)
 
     const paginationParams = this.getPaginationParams(req, pagedResults.page)
+    logger.debug('paginationParams', paginationParams)
```

Comment:
just checking if you meant to leave this in ?

Reply:
yes, for now - it's helping me do some testing - but it will come out - I'll add a TODO so I don't forget